opentelemetry/spring-boot-instrumentation/docker-compose.creds.yaml

```diff
@@ -26,8 +28,8 @@ services:
   otelcol:
     # If the collector does not have permission to read the mounted volumes, set
     # USERID=$(id -u) to run the container as the current user
-    user: ${USERID}
+    user: $USERID
```

Comment:
what is a reason for this change? To be syntactically correct the original line should be enclosed with quotes.

Reply:
The change is to be consistent with L35. Either style should be equivalent.

> To be syntactically correct the original line should be enclosed with quotes.

I'm not sure what you mean, what is wrong with this syntax?